client.go

```diff
@@ -748,7 +751,15 @@ func (sc *SnowthClient) DoRequestContext(ctx context.Context, node *SnowthNode,
 	}
 
 	cr := sc.ConnectRetries()
-	nodes := append([]*SnowthNode{node}, sc.ListActiveNodes()...)
+
+	// Create a randomized list of nodes to retry.
+	nodes := sc.ListActiveNodes()
+	rand.Shuffle(len(nodes), func(i, j int) {
```

Comment:
Rather than all this shuffling, wouldn't it be a lot quicker to just use node  `current_microsecond % len(nodes)` ?